### PR TITLE
Feature/mprc 5646 released money

### PR DESCRIPTION
**Pull request by ccamargov-meli**
## Description
Dev-site implementation for new `release-report` for all sites and translations. Includes implementation of dynamic domains for urls, using [FAKER][URL][DOMAIN]

### Issue involved
- [MPRC-5737](https://mercadolibre.atlassian.net/browse/MPRC-5737)

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/manage-account/reports/released-money/glossary.en.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+| STORE_ID | Branch ID if payment is made through a physical commerce. |
+| STORE_NAME | Branch name for payment made through a physical commerce. |
+| EXTERNAL_STORE_ID | User-defined branch ID for payment made through a physical commerce. |
+| ORDER_ID | Purchase Order. |
+| SHIPPING_ID | Shipping Identification. |
+| SHIPMENT_MODE | Shipping Mode. |
+| PACK_ID | Package identification in the cart. |
+| TAXES_DISAGGREGATED | Taxes disaggregated in JSON format. |
+
+> LEFT_BUTTON_RECOMMENDED_ES
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/glossary.en.md:38]:
```suggestion
> LEFT_BUTTON_RECOMMENDED_EN
```

File: guides/manage-account/reports/released-money/glossary.en.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+>
+> How to use this report
+>
+> Learn how the report is composed and how to analyze it to make your reconciliation. 
+>
+> [How to use this report](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/released-money/how-to-use/)
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/glossary.en.md:44]:
```suggestion
> [How to use this report](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/reports/released-money/how-to-use/)
```

File: guides/manage-account/reports/released-money/glossary.en.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+> RIGHT_BUTTON_REQUIRED_ES
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/glossary.en.md:46]:
```suggestion
> RIGHT_BUTTON_REQUIRED_EN
```

File: guides/manage-account/reports/released-money/glossary.en.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+>
+> Generate your report
+>
+> Learn the ways to generate a report and follow the steps to set your preferences.
+>
+> [Generate your report](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/released-money/generate/)
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/glossary.en.md:52]:
```suggestion
> [Generate your report](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/reports/released-money/generate/)
```

File: guides/manage-account/reports/released-money/glossary.pt.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+| TAXES_DISAGGREGATED | Impostos desagregados no formato JSON. |
+
+
+<hr/>
+
+### Próximos passos
+
+> LEFT_BUTTON_RECOMMENDED_PT
+>
+> Como usar o relatório
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/glossary.pt.md:45]: ¿Quedan diferentes?

File: guides/manage-account/reports/released-money/introduction.en.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+>
+> Note
+>
+> Have the [Releases report Glossary](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/released-money/glossary/) on hand to review it when needed or want to review a technical term.
+
+<hr/>
+
+### Next steps
+
+> LEFT_BUTTON_RECOMMENDED_ES
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.en.md:48]:
```suggestion
> LEFT_BUTTON_RECOMMENDED_EN
```

File: guides/manage-account/reports/released-money/introduction.en.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+>
+> How to use the report
+>
+> Learn the composition of the report and how to read it to make your reconciliation
+>
+> [How to use the report](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/released-money/how-to-use/)
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.en.md:54]:
```suggestion
> [How to use the report](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/reports/released-money/how-to-use/)
```

File: guides/manage-account/reports/released-money/introduction.en.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+> RIGHT_BUTTON_REQUIRED_ES
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.en.md:56]:
```suggestion
> RIGHT_BUTTON_REQUIRED_EN
```

File: guides/manage-account/reports/released-money/introduction.en.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+>
+> Generate your reports
+>
+> Learn the ways to generate a report and follow the steps to set your preferences.
+>
+> [Generate your reports](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/release-money/generate/)
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.en.md:62]:
```suggestion
> [Generate your reports](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/reports/release-money/generate/)
```

File: guides/manage-account/reports/released-money/introduction.es.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+> Para saber la fecha exacta en la que estará disponible el dinero de una operación, revisa el [detalle de tus cobros acreditados](https://www.mercadopago[FAKER][URL][DOMAIN]/activities/balance).
+
+
+## Para qué usar el reporte
+
+### Usa el reporte para conciliar
+
+**Este reporte se usa para conciliar** el dinero que existe en tu cuenta en un momento determinado e incluye las transacciones que componen a esos movimientos. 
+
+Genera el reporte de ----[mla]---- liquidaciones ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- liberaciones ------------ cada vez que necesites conciliar el detalle del dinero liberado entre dos fechas. Podrás saber cuándo se hizo cada transacción, en qué concepto y por qué monto, incluyendo los casos de bloqueo y desbloqueo.
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.es.md:31]:
```suggestion
Genera el reporte de ----[mla]---- Liquidaciones ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- Liberaciones ------------ cada vez que necesites conciliar el detalle del dinero liberado entre dos fechas. Podrás saber cuándo se hizo cada transacción, en qué concepto y por qué monto, incluyendo los casos de bloqueo y desbloqueo. 
```

File: guides/manage-account/reports/released-money/introduction.es.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
+# ¿Qué es el Reporte de ----[mla]---- Liquidaciones ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- Liberaciones ------------?
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.es.md:1]:
```suggestion
# ¿Qué es el reporte de ----[mla]---- Liquidaciones ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- Liberaciones ------------?
```

File: guides/manage-account/reports/released-money/introduction.pt.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
+# O que é o Relatório de ----[mla]---- Liquidaçoes ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- Liberações ------------?
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.pt.md:1]:
```suggestion
# O que é o relatório de ----[mla]---- Liquidaçoes ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- Liberações ------------?
```

File: guides/manage-account/reports/released-money/introduction.pt.md [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+>
+> Nota
+>
+> Tenha em mãos o [Glossário do relatório](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/released-money/glossary/) de dinheiro ----[mla]---- Liquidaçoes ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- Liberações ------------ para revisá-lo quando precisar ou caso queira consultar algum termo técnico.
+
+<hr/>
+
+### Próximos passos
+
+> LEFT_BUTTON_RECOMMENDED_ES
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.pt.md:47]:
```suggestion
> LEFT_BUTTON_RECOMMENDED_PT
```

File: guides/manage-account/reports/released-money/introduction.pt.md [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+>
+> Como usar o relatório           
+>
+> Saiba como o relatório é composto e aprenda a analisá-lo para fazer sua conciliação.
+>
+> [Como usar o relatório](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/released-money/how-to-use/)
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.pt.md:53]:
```suggestion
> [Como usar o relatório](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/manage-account/reports/released-money/how-to-use/)
```

File: guides/manage-account/reports/released-money/introduction.pt.md [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+
+> RIGHT_BUTTON_REQUIRED_ES
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.pt.md:55]:
```suggestion
> RIGHT_BUTTON_REQUIRED_PT
```

File: guides/manage-account/reports/released-money/introduction.pt.md [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+>
+> Gere seus relatórios
+>
+> Saiba as formas de gerar um relatório e siga as etapas para configurar suas preferências.
+>
+> [Gere seus relatórios](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/release-money/generate/)
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.pt.md:61]:
```suggestion
> [Gere seus relatórios](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/manage-account/reports/release-money/generate/)
```

File: guides/manage-account/reports/released-money/how-to-use.pt.md [hunk 17]
```diff
@@ -0,0 +1,83 @@
...
+> Importante: diferenças entre retirada parcial e retirada total.
+>
+> Quando você retirar todo o seu saldo disponível, o total do relatório corresponderá a esse valor. Por outro lado, quando você faz uma retirada parcial, que não inclui todo o seu dinheiro liberado na conta, o saldo total disponível e o total do relatório não coincidem.
+>
+>Por exemplo, imagine que você tenha R$ 3.000 disponíveis para retirar para uma conta bancária, mas retira apenas R$ 2.000. A retirada é parcial, mas o valor total do relatório continuará a mostrar o valor do saldo inicial que estava no momento da retirada, ou seja, os R$ 3.000 disponíveis. Por outro lado, se você retirar os R$ 3.000, o valor total do relatório corresponderá ao valor dessa retirada.
+
+
+<hr/>
+
+### Próximos passos
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/how-to-use.pt.md:67]:
```suggestion
---

### Próximos passos
```

File: guides/manage-account/reports/released-money/generate.es.md [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+
+
+> NOTE
+>
+> Nota
+>
+> Ten a mano el [Glosario del reporte](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/release-money/glossary/) de ----[mla]---- liquidaciones ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- liberaciones ------------ para revisarlo cuando lo necesites o quieras consultar algún término técnico.
+
+<hr/>
+
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/generate.es.md:50]:
```suggestion
---

```

File: guides/manage-account/reports/released-money/generate.en.md [hunk 19]
```diff
@@ -0,0 +1,69 @@
...
+
+> NOTE
+>
+> Note
+>
+> Have the [Releases report Glossary](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/released-money/glossary/) on hand to review it when needed or want to review a technical term.
+
+
+<hr/>
+
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/generate.en.md:52]:
```suggestion
---

```

File: guides/manage-account/reports/released-money/generate.pt.md [hunk 20]
```diff
@@ -0,0 +1,67 @@
...
+
+
+> NOTE
+>
+> Nota
+>
+> Tenha em mãos o [Glossário do relatório](https://www.mercadopago.com.br/developers/es/guides/manage-account/reports/released-money/glossary/) de ----[mla]---- liquidaçoes ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- liberações ------------ para consultá-lo quando precisar ou queira conferir algum termo técnico.
+
+<hr/>
+
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/generate.pt.md:50]:
```suggestion
---

```

File: guides/manage-account/reports/released-money/glossary.en.md [hunk 21]
```diff
@@ -0,0 +1,52 @@
...
+| EXTERNAL_POS_ID | User-defined cashier ID for payment made through a physical commerce. |
+| STORE_ID | Branch ID if payment is made through a physical commerce. |
+| STORE_NAME | Branch name for payment made through a physical commerce. |
+| EXTERNAL_STORE_ID | User-defined branch ID for payment made through a physical commerce. |
+| ORDER_ID | Purchase Order. |
+| SHIPPING_ID | Shipping Identification. |
+| SHIPMENT_MODE | Shipping Mode. |
+| PACK_ID | Package identification in the cart. |
+| TAXES_DISAGGREGATED | Taxes disaggregated in JSON format. |
+
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/glossary.en.md:37]:
```suggestion
---

### Next steps

```

File: guides/manage-account/reports/released-money/glossary.pt.md [hunk 22]
```diff
@@ -0,0 +1,57 @@
...
+| EXTERNAL_STORE_ID | ID da lojal definido pelo usuário para o pagamento feito em um comércio físico. |
+| ORDER_ID | Ordem de compra. |
+| SHIPPING_ID | Identificador de envio. |
+| SHIPMENT_MODE | Modalidade de envio. |
+| PACK_ID | Identificador do pacote no carrinho. |
+| TAXES_DISAGGREGATED | Impostos desagregados no formato JSON. |
+
+
+<hr/>
+
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/glossary.pt.md:40]:
```suggestion
---

```

File: guides/manage-account/reports/released-money/glossary.es.md [hunk 23]
```diff
@@ -0,0 +1,56 @@
...
+| STORE_NAME | Nombre de sucursal para el pago realizado a través de un comercio físico. |
+| EXTERNAL_STORE_ID | ID de sucursal definido por el usuario para el pago realizado a través de un comercio físico. |
+| ORDER_ID | Orden de compra. |
+| SHIPPING_ID | Identificador de envío. |
+| SHIPMENT_MODE | Modalidad de envío. |
+| PACK_ID | Identificador del paquete en el carrito. |
+| TAXES_DISAGGREGATED | Impuestos desagregados en formato JSON. |
+
+<hr/>
+
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/glossary.es.md:39]:
```suggestion
---

```

File: guides/manage-account/reports/released-money/how-to-use.en.md [hunk 24]
```diff
@@ -0,0 +1,76 @@
...
+> WARNING
+>
+> Important: differences between partial withdrawal and total withdrawal.
+>
+> When you withdraw all your available money, the total report will match that amount. On the other hand, when you make a partial withdrawal, which does not include all of your released account money, the total available balance and the total of the report will not match.
+>
+> For example, imagine that you have $ 3,000 available to withdraw to a bank account but only withdraw $ 2,000. The withdrawal is partial but the total value of the report will continue to show you the amount of the initial balance that was at the time of withdrawal, that is, the $ 3,000 you had available. On the other hand, if you withdraw the $ 3,000, the total value of the report will match the value of that withdrawal.
+
+<hr/>
+
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/how-to-use.en.md:59]:
```suggestion
---

```

File: guides/manage-account/reports/released-money/how-to-use.es.md [hunk 25]
```diff
@@ -0,0 +1,72 @@
...
+------------
+----[mlm, mlb, mlc, mco, mlu, mpe]----
+![Reporte de liberaciones](/images/manage-account/reports/example-release-es.jpg)
+------------
+
+La versión por defecto mostrará una vista extendida de las columnas. El informe final tendrá la mayor cantidad de detalle posible.
+
+
+<hr/>
+
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/how-to-use.es.md:55]:
```suggestion
---

```

File: guides/manage-account/reports/released-money/how-to-use.pt.md [hunk 26]
```diff
@@ -0,0 +1,83 @@
...
+>
+> Importante: diferenças entre retirada parcial e retirada total.
+>
+> Quando você retirar todo o seu saldo disponível, o total do relatório corresponderá a esse valor. Por outro lado, quando você faz uma retirada parcial, que não inclui todo o seu dinheiro liberado na conta, o saldo total disponível e o total do relatório não coincidem.
+>
+>Por exemplo, imagine que você tenha R$ 3.000 disponíveis para retirar para uma conta bancária, mas retira apenas R$ 2.000. A retirada é parcial, mas o valor total do relatório continuará a mostrar o valor do saldo inicial que estava no momento da retirada, ou seja, os R$ 3.000 disponíveis. Por outro lado, se você retirar os R$ 3.000, o valor total do relatório corresponderá ao valor dessa retirada.
+
+
+<hr/>
+
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/how-to-use.pt.md:66]:
```suggestion
---

```

File: guides/manage-account/reports/released-money/introduction.en.md [hunk 27]
```diff
@@ -0,0 +1,62 @@
...
+In addition, you can use the report to know the history of releases between two dates or two withdrawals, the detail of the events that make up an automatic or manual withdrawal (total or partial), the detail of disputes, refunds, shipments, chargebacks , taxes and other operations that affect the available balance and the detail of the installments released and the ones that are still to be paid.
+
+> NOTE
+>
+> Note
+>
+> Have the [Releases report Glossary](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/released-money/glossary/) on hand to review it when needed or want to review a technical term.
+
+<hr/>
+
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.en.md:45]:
```suggestion
---

```

File: guides/manage-account/reports/released-money/introduction.es.md [hunk 28]
```diff
@@ -0,0 +1,62 @@
...
+Además, puedes usar el reporte para conocer el historial de dinero ----[mla]---- liquidado ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- liberado ------------ entre dos fechas, el detalle de los eventos que componen una transferencia bancaria, el detalle de disputas, reembolsos, envíos, contracargos, impuestos y otras operaciones. También verás el detalle de las cuotas liquidadas y las cuotas que aún faltan saldar.
+
+> NOTE
+>
+> Nota
+>
+> Ten a mano el [Glosario del reporte](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/released-money/glossary/) de ----[mla]---- liquidaciones ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- liberaciones ------------ para revisarlo cuando lo necesites o quieras consultar algún término técnico.
+
+<hr/>
+
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.es.md:45]:
```suggestion
---

```

File: guides/manage-account/reports/released-money/introduction.pt.md [hunk 29]
```diff
@@ -0,0 +1,61 @@
...
+Além disso, você pode usar o relatório para saber o histórico de dinheiro ----[mla]---- liquidado ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- liberado ------------ entre duas datas, o detalhe dos eventos que compõem uma transferência bancária, os detalhes de disputas, reembolsos, envios, estornos, impostos e outras transações. Você também verá os detalhes das parcelas liquidadas e as parcelas ainda a serem pagas.
+
+> NOTE
+>
+> Nota
+>
+> Tenha em mãos o [Glossário do relatório](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/reports/released-money/glossary/) de dinheiro ----[mla]---- Liquidaçoes ------------ ----[mlm, mlb, mlc, mco, mlu, mpe]---- Liberações ------------ para revisá-lo quando precisar ou caso queira consultar algum termo técnico.
+
+<hr/>
+
```
Comment (pamalv7) [inline guides/manage-account/reports/released-money/introduction.pt.md:44]:
```suggestion
---

```

**Review by pamalv7 — CHANGES_REQUESTED** [review]

File: guides/manage-account/reports/released-money/generate.en.md [hunk 30]
```diff
@@ -0,0 +1,69 @@
...
+Know the characteristics of the elements that make up your report.
+
+
+| Element or action | Characteristics |
+| ------------ |	--------    |
+| Tables Detail | <br/>The detail of the tables includes information generated in at least 1 day. <br/> <br/>  |
+| Column Order |<br/> Fix <br/> <br/> |
+| Maximum Period | <br/> Reports with data of up to 60 days <br/> <br/> |
+| Currency | <br/> Local (basada en el país donde esté registrada la cuenta de Mercado Pagobased on the country where the Mercado Pago account is registered) <br/> <br/> |
```
[the remark this answers is not in the document]
Comment (ccamargov-meli) [reply guides/manage-account/reports/released-money/generate.en.md:27]: Está en inglés y en español, procedo a eliminar la parte en español.

**Review by ccamargov-meli — COMMENTED** [review]
Done.

**Review by pamalv7 — APPROVED** [review]

**Pull request merged**